services/qs-spark-service/src/main/java/ai/quantumics/api/util/RuleJobHelper.java

```diff
@@ -123,11 +147,32 @@ private String rowCountEtlScriptVarsInit(
         temp = temp.replace(SOURCE_PATH, String.format("'%s'", ruleDetails.getSourceData().getFilePath()));
         temp = temp.replace(TARGET_BUCKET, String.format("'%s'", ruleDetails.getTargetData().getBucketName()));
```

Comment:
what happens if it is only for source and no target bucket is available? . I think this line will throw null exception